plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -153,6 +154,10 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
         showMinLabel: xAxisShowMinLabel,
         showMaxLabel: xAxisShowMaxLabel,
         formatter: xAxisFormatter,
+        rotate:
+          xAxisLabelRotation >= -90 && xAxisLabelRotation <= 90
+            ? xAxisLabelRotation
+            : DEFAULT_FORM_DATA.xAxisLabelRotation,
```

Comment:
I think we can remove this - let's do as much validation as possible in the control. I think validating for numeric values is enough at this point (in reference to the previous comment).

Reply:
We should probably allow other degrees if ECharts supports it. I'm always in favor of not adding hurdles to strange user behaviors unless absolutely necessary.